README.md

```diff
@@ -24,8 +24,8 @@ and being able to run that code on any cloud you want, avoiding all the ceremony
 of cloud-specific authorization, tracing, SDKs and all the other code required
 to make an application portable across cloud platforms.
 
-Another key part of the project is a code generator called
-[Wire](https://github.com/google/go-cloud/blob/master/wire/README.md). It
+The project works with a code generator called
```

Comment:
Maybe "works well with"?
I don't think that we want to imply that it requires Wire.

Reply:
Done.